modules/ossm-cert-manager-verify.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-security.adoc
+
+
+:_content-type: PROCEDURE
+[id="ossm-cert-manager-verify_{context}"]
+== Verify cert-manager installation
+
+To verify cert-manager is installed, follow these steps:
+
+. Check `istiod` certificate:
++
+[source, terminal]
+----
+kubectl exec $(kubectl get pods -l app=productpage -o jsonpath='{.items[0].metadata.name}') -c istio-proxy -- \
+    openssl s_client -showcerts istiod-basic.istio-system:15012 < /dev/null
+----
+
+. Check app certificate:
++
+[source, terminal]
+----
+kubectl exec $(kubectl get pods -l app=productpage -o jsonpath='{.items[0].metadata.name}') -c istio-proxy -- \
+    openssl s_client -showcerts details.bookinfo:9080 < /dev/null
+----
+//Waiting on Jacek/devs to clarify what "mounts" and "root configmap" and "root of trust" mean
+. Verify by seeing all mounts are correct on pilot, launching bookinfo and checking.
+
+. Verify that the right root config map is used for the root of trust.
```

Comment:
Can we add here the command or from where we validate this?

Reply:
@gwynnemonahan We can add this example command? We put the expected output here but we do not explain from where comes this output